access_point/wap.sh

```diff
@@ -56,10 +56,7 @@ echo "denyinterfaces wlan0" >> /etc/dhcpcd.conf
 systemctl enable hostapd
 systemctl enable dnsmasq
 
-sudo service hostapd start
```

Comment:
@hongquan , why have you not started and added to be done on a later stage

Reply:
Sorry, I don't very understand what you mean, @stealthanthrax .

Here, I remove the line `sudo service hostapd start`, because this script will make RPI to reboot, so no need to start `hostapd` here.